src/olympia/shelves/models.py

```diff
@@ -50,6 +56,20 @@ def get_count(self):
             3 if self.addon_type == amo.ADDON_STATICTHEME else 4
         )
 
+    @cached_property
```

Comment:
What are the implications of this being a `@cached_property`? Are we still always going to get a new, random tag each time the API is called? Or maybe I don't understand exactly how this property is used.

Reply:
`cached_property` stores the value on the object instance - i.e. in this case the `Shelf` object.  `ShelfViewSet` creates the queryset (which is an iterable of `Shelf` objects) each time the API is called so the `cached_property` value will only persist for the life cycle of the API request.